Controller/app-controller.js

```diff
@@ -0,0 +1,7 @@
+const {selectTopics} = require('../Model/app-model')
+
+exports.getTopics = (req, res) => {
+    selectTopics().then(results => {
+        res.status(200).send(results)
+    })
+}
```

Comment:
Here you are sending back the topics as an array, which doesn't follow json convention. Make sure to send back all responses in an object with the key describing the data found at the value. For example, I would want this data to be sent back as the following

```js
{ topics: results }
```